docs/diagrams/BatchCommandmatchInputStringToIndexSequenceDiagram.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
+participant Actor
```

Comment:
I would suggest importing the [styles.puml](https://github.com/AY2122S2-CS2103-W17-1/tp/blob/37ab69eebea151008dd755c88d5ccb895064a68e/docs/diagrams/style.puml) file to ensure the diagram styles are consistent with those in the DG. You can take reference to the sequence diagram files to see how to group the various classes together.